opentelemetry-api/tests/context/test_context.py

```diff
@@ -74,3 +77,14 @@ def test_set_current(self):
 
         context.detach(token)
         self.assertEqual("yyy", context.get_value("a"))
+
+
+class TestInitContext(unittest.TestCase):
+    def test_load_runtime_context_default(self):
+        ctx = context._load_runtime_context()  # pylint: disable=W0212
+        self.assertIsInstance(ctx, ContextVarsRuntimeContext)
+
+    @patch.dict("os.environ", {OTEL_PYTHON_CONTEXT: "contextvars_context"})
```

Comment:
Could you add a new test case for the special fallback case we discussed (the case where `OTEL_PYTHON_CONTEXT` does not have a matching entrypoint)

Reply:
Added